pkg/config/nodetreemodel/config.go

```diff
@@ -420,16 +464,7 @@ func (c *ntmConfig) AllKeysLowercased() []string {
 	return slices.Clone(c.allSettings)
 }
 
-func (c *ntmConfig) leafAtPath(key string) LeafNode {
-	return c.leafAtPathFromNode(key, c.root)
-}
-
 func (c *ntmConfig) leafAtPathFromNode(key string, curr Node) LeafNode {
```

Comment:
Is this called anywhere using a value for `curr` that isn't equal to `c.root`?

Reply:
Not right now, I moved this method to the getter file where it's used.